src/lib/gundb/UserStorageClass.js

```diff
@@ -2368,7 +2368,10 @@ export class UserStorage {
    * @returns {Promise<boolean>|Promise<boolean>}
    */
   async userAlreadyExist(): Promise<boolean> {
-    const exists = await this.gunuser.get('registered').then(null, 1000)
+    // from GUN docs: gun.then(cb) (not official yet) Could be buggy until official!
```

Comment:
revert this

Reply:
done. Also i've fixed .then override - you're always passed { wait: 200 } despite to the 'wait' param value